dss/config.py

```diff
@@ -348,6 +350,35 @@ def debug_level() -> int:
         """
         return int(os.environ.get('DSS_DEBUG', '0'))
 
+    @staticmethod
+    def get_openid_provider():
+        return Config._get_required_envvar("OPEN_ID_PROVIDER")
```

Comment:
I still think OPENID should be one word.

Reply:
fixed